src/cf-pagination/src/molecules/pagination.less

```diff
@@ -0,0 +1,107 @@
+//
+// Theme variables
+//
+
+// Color variables
+// $color- variables are from 18F's US Web Design Standards
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
+
+@pagination-text-color: #757575; // $color-gray-medium
+@pagination-bg-color: #f1f1f1; // $color-gray-lightest
+
+// Sizing variables
+
+@pagination-font-size__px: @base-font-size-px;
+@pagination-font-size__em: unit( @pagination-font-size__px / @base-font-size-px, em );
+@pagination-height__px: 42px;
+@pagination-width__px: 130px;
+
+//
+// Pagination
+//
+
+.m-pagination {
+    position: relative;
+
+    &_form {
```

Comment:
It's not a container for everything, only form elements in the middle, so I don't think it could be a div because the "Go" button would stop working. The buttons are absolutely positioned on top of the form.